mix.exs

```diff
@@ -33,7 +33,16 @@ defmodule Membrane.MP4.Plugin.MixProject do
       main: "readme",
       extras: ["README.md"],
```

Comment:
Please get rid of warning during docs generation: move `LICENSE.md` to `LICENSE` (fixes rendering in docs) and then add:
```suggestion
      extras: ["README.md", "LICENSE"],
```